### PR TITLE
Add OrgBook case study and implement layout changes for case studies

### PR DESCRIPTION
**Pull request by mark-a-wilson**
This PR implements #880. Summary:

- Adds content for a new case study about OrgBook BC
- Changes style of case study cards from the flat style to clickable cards
- Changes layout of cards that appear on the list page and at the bottom of each detail page ("Other Case Studies")

### CODE REVIEW

File: react-frontend/src/components/Community/communityCard.js [hunk 1]
```diff
@@ -11,7 +12,9 @@ function CommunityCard({ title, description, uid }) {
   return (
     <CommunityCardStyled to={`/communities/${uid}`}>
       <CommunityCardHeader>{title}</CommunityCardHeader>
-      <CommunityCardDescription>{description}</CommunityCardDescription>
+      <CardClickableBody>
```
Comment (thegentlemanphysicist) [inline react-frontend/src/components/Community/communityCard.js:15]: Why is this being changed?  I don't see it being referenced in the zenhub QA cards or in the PR description

File: react-frontend/src/constants/urls.js [hunk 2]
```diff
@@ -130,6 +130,8 @@ export const resourceLinks = {
     'https://vancouversun.com/business/local-business/b-c-announces-more-changes-resulting-from-mount-polley-mine-failure',
   news3:
     'https://bc.ctvnews.ca/mobile/british-columbia-proposes-reforms-to-strengthen-oversight-of-mining-industry-1.4605156?cache=yes?clipId=263414',
+  nutrientCalculator: 'https://nmp.apps.nrs.gov.bc.ca',
+  orgBookBC: 'https://orgbook.gov.bc.ca/en/home',
```
Comment (thegentlemanphysicist) [inline react-frontend/src/constants/urls.js:134]: Good use of urls

File: react-frontend/src/components/StyleComponents/card.js [hunk 3]
```diff
@@ -29,16 +31,20 @@ export const CardClickable = styled(Link).attrs({
   }
 `;
 
+export const CardClickableBody = styled.div.attrs({
```
Comment (thegentlemanphysicist) [inline react-frontend/src/components/StyleComponents/card.js:34]: Restyling these cards has now broken common components cards (please see deployment)

**Review by thegentlemanphysicist — CHANGES_REQUESTED** [review]
The card restyling has caused the common component card styling to break.

File: react-frontend/src/components/StyleComponents/card.js (hunk 3 above)
Comment (mark-a-wilson) [reply react-frontend/src/components/StyleComponents/card.js:34]: Good catch, thank you. Please see https://github.com/bcgov/digital.gov.bc.ca/pull/955/commits/9766faed283b7dc67cff18be97f25796e7b344b1

File: react-frontend/src/components/Community/communityCard.js (hunk 1 above)
Comment (mark-a-wilson) [reply react-frontend/src/components/Community/communityCard.js:18]: Idea here was to introduce a **body** across all cards that would define the 24px padding and rounded bottom corners in one place, into which various other items can fit (e.g. titles, subtitles, descriptions, avatars, tags) based on the variations. 

I've manually tested card styles across the site and nothing is broken now. 

In a future, separate issue, I'd like to tackle a refactor of card components -- I think the number of styles can be reduced.

**Review by thegentlemanphysicist — APPROVED** [review]
Looks good, I wasn't able to find any breaking changes.  :)

**Pull request merged**